### ISSUE TITLE
Consistent argument names for video datasets and VideoClips class

### ISSUE THREAD

**Issue by MrinalJain17** [pytorch/vision#1227]
First of all, the new support for Video data is amazing. Really useful. 😃 

Just when I was going through the code of the video datasets and utilities, I found this a bit confusing.

Sample Dataset:
https://github.com/pytorch/vision/blob/8635be94d1216f10fb8302da89233bd86445e449/torchvision/datasets/ucf101.py#L27-L36

`VideoClips` class: 
https://github.com/pytorch/vision/blob/8635be94d1216f10fb8302da89233bd86445e449/torchvision/datasets/video_utils.py#L43-L49

From what I understand, the argument `frames_per_clip` from the dataset is the same as `clip_length_in_frames` from the `VideoClips` class. 

Similarly, `step_between_clips` <-> `frames_between_clips`.

Is there a reason for this difference. Or did I understand it wrong?

Would it be better if both, the video datasets and the `VideoClips` class had consistent arguments? 

In my opinion, `frames_per_clip` looks better than `clip_length_in_frames`.
And `frames_between_clips` can be favored over `step_between_clips` since the former feels more intuitive.

Any suggestions? @fmassa

**Comment by fmassa** [comment]
Good catch, having different names is indeed not great and we should strive to have a single name.

I should have unified those naming differences before the release, but I didn't know which one to pick and I ended up forgetting.

I'm yet unsure about which names are better though. @bjuncek do you have any opinions here?

**Comment by bjuncek** [comment]
I think `clip_length_in_frames` and `frames_between_clips` make more sense, but I'm open for discussion :)

**Comment by fmassa** [comment]
So, from the current discussion, @bjuncek prefers the names that are currently used in `VideoClips`. I like them because they are very explicit, but they are a bit too long maybe.

One action point would then be to modify the 3 datasets (UCF101, Kinetics400 and HMDB51) to take the same names as `VideoClips`, while keeping proper deprecation warning for the old names.

Thoughts?

**Comment by bjuncek** [comment]
Sounds good to me.

**Comment by ekosman** [comment]
I find it hard to understand what frame_rate means. At first I thought it means fps, however in the init function of VideoClips, frame_rate is passed to "compute_clips" and the documentations for this function names the 3rd parameter "dilation":
```
Arguments:
            num_frames (int): number of frames for the clip
            step (int): distance between two clips
            dilation (int): distance between two consecutive frames
                in a clip
        """
```

If I want to sample every second frame, i.e. 32 frames in the original video become 1 clip of 16 frames, what should frame_rate be?
